.pre-commit-config.yaml

```diff
@@ -65,6 +65,7 @@ repos:
     rev: v3.2.0  # Use the ref you want to point at
     hooks:
     -   id: check-added-large-files
```

Comment:
there's a way to set two thresholds, one for H5 files and one for everything else, see the sdk.  that way we still have some threshold in place for H5 files